rackspace/autoscale/v1/policies/requests.go

```diff
@@ -0,0 +1,251 @@
+package policies
+
+import (
+	"errors"
+
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// Validation errors returned by create or update operations.
+var (
+	ErrNoName            = errors.New("Policy name cannot by empty.")
+	ErrNoArgs            = errors.New("Args cannot be nil for schedule policies.")
+	ErrInvalidAdjustment = errors.New("Invalid adjustment type.")
+)
+
+// List returns all scaling policies for a group.
+func List(client *gophercloud.ServiceClient, groupID string) pagination.Pager {
+	url := listURL(client, groupID)
+
+	createPageFn := func(r pagination.PageResult) pagination.Page {
+		return PolicyPage{pagination.SinglePageBase(r)}
+	}
+
+	return pagination.NewPager(client, url, createPageFn)
+}
+
+// CreateOptsBuilder is the interface responsible for generating the map that
+// will be marshalled to JSON for a Create operation.
+type CreateOptsBuilder interface {
+	ToPolicyCreateMap() ([]map[string]interface{}, error)
+}
+
+// Adjustment represents the change in capacity associated with a policy.
+type Adjustment struct {
+	// The type for this adjustment.
+	Type AdjustmentType
+
+	// The value of the adjustment.  For adjustments of type Change or
+	// DesiredCapacity, this will be converted to an integer.
+	Value float64
+}
+
+// AdjustmentType represents the way in which a policy will change a group.
+type AdjustmentType string
+
+// Valid types of adjustments for a policy.
+const (
+	Change          AdjustmentType = "change"
+	ChangePercent   AdjustmentType = "changePercent"
+	DesiredCapacity AdjustmentType = "desiredCapacity"
+)
+
+// CreateOpts is a slice of CreateOpt structs that allow the user to create
+// multiple policies in a single operation.
+type CreateOpts []CreateOpt
+
+// CreateOpt represents the options to create a policy.
+type CreateOpt struct {
+	// Name [required] is a name for the policy.
+	Name string
+
+	// Type [required] of policy, i.e. either "webhook" or "schedule".
+	Type Type
+
+	// Cooldown [required] period in seconds.
+	Cooldown int
+
+	// Adjustment [requried] type and value for the policy.
+	Adjustment Adjustment
+
+	// Additional configuration options for some types of policy.
+	Args map[string]interface{}
```

Comment:
The latter. We'll want to be as specific as we can so that users know what their options are, and so that they don't have to worry about "cron" vs "CRON" vs "Cron" etc. Also, we can force the correct time format on users for the `At` field. Something like:
```go
type ScheduleType string
var (
  Cron ScheduleType = "cron"
  At ScheduleType = "at"
)

type CreateOpt struct {
//...
ScheduleType ScheduleType
ScheduleValue string
//...
}
```

That's ^ just an idea. There's an argument to made for defining it differently in such a way as to have the value for `At` be a `time.Time`, but having it as a string let's us check the format easily with `time.Parse`.